### PR TITLE
[docs] Add commands table to the README

### LINKED ISSUE #147: [feature] Create a table with all possible bot commands for reference
### Is your feature request related to a problem? Please describe

Many Discord bots have this e.g. https://github.com/brendacs/stop-bot

### Additional context

We have a lot of commands now and should have some way to easily let people know what they are / reference them.

### PR DESCRIPTION
**Pull request by rajpiskala**
### Purpose of pull request

**Related Issue**: resolves #147

Adds a command table with all commands for the bots to the README. Useful as a reference for anyone who wants to know commands the bot has.

### Pull request checklist

- [x] Branch and PR are named correctly
- [x] Updated relevant documentation
- [x] Tested with a tester bot
- [x] Wrote unit tests for changes

### Testing instructions

You can visually confirm the contents of the command table by going to my branch. You can directly modify the contents of the table in the source code in README.md to change it to your liking.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,51 @@
...
+|   :safetypin: (reaction)   |            Pin a message to the server channel            | Admin, Mod, Contributor |
+|        **Settings**        |                                                           |                         |
+|  `purge [delete \| lock]`  |      Deletes all messages from or locks the channel       |          Admin          |
+|      `setDevChannel`       |            Sets channel as a developer channel            |          Admin          |
+|     `setEventChannel`      |             Sets channel as an event channel              |          Admin          |
+|      `setModChannel`       |               Sets channel as a mod channel               |          Admin          |
+| `update VAR_NAME <value>`  | Updates environment variable and refreshes env to take it |          Admin          |
+|          **Misc**          |                                                           |                         |
+|      `lmgtfy <query>`      |            Creates a lmgtfy link for the query            |        Everyone         |
+| `pm @<username> <message>` |                  Direct messages a user                   |        Everyone         |
```
Comment (nikgil) [inline README.md]: Mod

File: README.md [hunk 2]
```diff
@@ -6,6 +6,51 @@
...
+|           `list`           |       Messages you with list of all pinned messages       | Admin, Mod, Contributor |
+|   :safetypin: (reaction)   |            Pin a message to the server channel            | Admin, Mod, Contributor |
+|        **Settings**        |                                                           |                         |
+|  `purge [delete \| lock]`  |      Deletes all messages from or locks the channel       |          Admin          |
+|      `setDevChannel`       |            Sets channel as a developer channel            |          Admin          |
+|     `setEventChannel`      |             Sets channel as an event channel              |          Admin          |
+|      `setModChannel`       |               Sets channel as a mod channel               |          Admin          |
+| `update VAR_NAME <value>`  | Updates environment variable and refreshes env to take it |          Admin          |
+|          **Misc**          |                                                           |                         |
+|      `lmgtfy <query>`      |            Creates a lmgtfy link for the query            |        Everyone         |
```
Comment (nikgil) [inline README.md]: Mod

File: README.md [hunk 3]
```diff
@@ -6,6 +6,51 @@
...
+|            `mc`            |     Alerts moderators with anonymous private message      |        Everyone         |
+|          **Pins**          |                                                           |                         |
+|           `list`           |       Messages you with list of all pinned messages       | Admin, Mod, Contributor |
+|   :safetypin: (reaction)   |            Pin a message to the server channel            | Admin, Mod, Contributor |
+|        **Settings**        |                                                           |                         |
+|  `purge [delete \| lock]`  |      Deletes all messages from or locks the channel       |          Admin          |
+|      `setDevChannel`       |            Sets channel as a developer channel            |          Admin          |
+|     `setEventChannel`      |             Sets channel as an event channel              |          Admin          |
+|      `setModChannel`       |               Sets channel as a mod channel               |          Admin          |
+| `update VAR_NAME <value>`  | Updates environment variable and refreshes env to take it |          Admin          |
```
Comment (nikgil) [inline README.md]: Mod

File: README.md [hunk 4]
```diff
@@ -6,6 +6,51 @@
...
+|        **Modchat**         |                                                           |                         |
+|            `mc`            |     Alerts moderators with anonymous private message      |        Everyone         |
+|          **Pins**          |                                                           |                         |
+|           `list`           |       Messages you with list of all pinned messages       | Admin, Mod, Contributor |
+|   :safetypin: (reaction)   |            Pin a message to the server channel            | Admin, Mod, Contributor |
+|        **Settings**        |                                                           |                         |
+|  `purge [delete \| lock]`  |      Deletes all messages from or locks the channel       |          Admin          |
+|      `setDevChannel`       |            Sets channel as a developer channel            |          Admin          |
+|     `setEventChannel`      |             Sets channel as an event channel              |          Admin          |
+|      `setModChannel`       |               Sets channel as a mod channel               |          Admin          |
```
Comment (nikgil) [inline README.md]: Mod

File: README.md [hunk 5]
```diff
@@ -6,6 +6,51 @@
...
+|           `list`           |        Show all the scheduled AMAs for the future         |          Admin          |
+|        **Modchat**         |                                                           |                         |
+|            `mc`            |     Alerts moderators with anonymous private message      |        Everyone         |
+|          **Pins**          |                                                           |                         |
+|           `list`           |       Messages you with list of all pinned messages       | Admin, Mod, Contributor |
+|   :safetypin: (reaction)   |            Pin a message to the server channel            | Admin, Mod, Contributor |
+|        **Settings**        |                                                           |                         |
+|  `purge [delete \| lock]`  |      Deletes all messages from or locks the channel       |          Admin          |
+|      `setDevChannel`       |            Sets channel as a developer channel            |          Admin          |
+|     `setEventChannel`      |             Sets channel as an event channel              |          Admin          |
```
Comment (nikgil) [inline README.md]: Mod

File: README.md [hunk 6]
```diff
@@ -6,6 +6,51 @@
...
+|           `info`           |           Show information about the AMA event            |          Admin          |
+|           `list`           |        Show all the scheduled AMAs for the future         |          Admin          |
+|        **Modchat**         |                                                           |                         |
+|            `mc`            |     Alerts moderators with anonymous private message      |        Everyone         |
+|          **Pins**          |                                                           |                         |
+|           `list`           |       Messages you with list of all pinned messages       | Admin, Mod, Contributor |
+|   :safetypin: (reaction)   |            Pin a message to the server channel            | Admin, Mod, Contributor |
+|        **Settings**        |                                                           |                         |
+|  `purge [delete \| lock]`  |      Deletes all messages from or locks the channel       |          Admin          |
+|      `setDevChannel`       |            Sets channel as a developer channel            |          Admin          |
```
Comment (nikgil) [inline README.md]: Mod

File: README.md [hunk 7]
```diff
@@ -6,6 +6,51 @@
...
+|           `edit`           |             Edit AMA information (time, info)             |          Admin          |
+|           `info`           |           Show information about the AMA event            |          Admin          |
+|           `list`           |        Show all the scheduled AMAs for the future         |          Admin          |
+|        **Modchat**         |                                                           |                         |
+|            `mc`            |     Alerts moderators with anonymous private message      |        Everyone         |
+|          **Pins**          |                                                           |                         |
+|           `list`           |       Messages you with list of all pinned messages       | Admin, Mod, Contributor |
+|   :safetypin: (reaction)   |            Pin a message to the server channel            | Admin, Mod, Contributor |
+|        **Settings**        |                                                           |                         |
+|  `purge [delete \| lock]`  |      Deletes all messages from or locks the channel       |          Admin          |
```
Comment (nikgil) [inline README.md]: Mod

File: README.md [hunk 8]
```diff
@@ -6,6 +6,51 @@
...
 
+## Commands
+
+**Default prefix:** `++`
+
+**Command types:**
+
+|    Type    |                                                  Description                                                  |
+| ---------- | ------------------------------------------------------------------------------------------------------------- |
+|   `ama`    |                              Used to queue or collect questions for AMA events.                               |
```
Comment (nikgil) [inline README.md]: this is more expansive than just questions for events, but a lot of it is still in dev

Comment (rajpiskala) [reply README.md]: I took the descriptions for each command category from /specs, so I'll update it in both the table and in the spec sheet.

**Review by nikgil — APPROVED** [review]

**Pull request merged**